src/main/view/game/StructureOverviewPanel.java

```diff
@@ -1,33 +1,219 @@
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.input.ScrollEvent;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
 import view.GameModelAdapter;
 import view.ViewEnum;
 import view.assets.AssetManager;
 
-public class StructureOverviewPanel extends OverviewPanel {
-	public StructureOverviewPanel(GameModelAdapter gameModelAdapter, AssetManager assets,
-			ViewEnum view) {
+public class StructureOverviewPanel extends OverviewPanel{
+	private TableView<StructureItem> structureTable = new TableView<>();
+	private Label label = new Label("structure Overview");
+	private TableColumn<StructureItem, String> structureTypeColumn = new TableColumn<>("Type");
+	private TableColumn<StructureItem, String> healthColumn = new TableColumn<>("Health");
+	private TableColumn<StructureItem, String> attackColumn = new TableColumn<>("Attack");
+	private TableColumn<StructureItem, String> defenseColumn = new TableColumn<>("Defense");
+	private TableColumn<StructureItem, String> armorColumn = new TableColumn<>("Armor");
+	private TableColumn<StructureItem, String> upkeepColumn = new TableColumn<>("Upkeep");
+	private TableColumn<StructureItem, String> workerColumn = new TableColumn<>("Workers");
+	private ObservableList<StructureItem> structureList = FXCollections.observableArrayList();
+	private VBox structureBox = new VBox();
+	private DropShadow ds = new DropShadow();
+	private ScrollPane scrollPane = new ScrollPane();
+	private Group root;
+	
+	public StructureOverviewPanel(GameModelAdapter gameModelAdapter, AssetManager assets, ViewEnum view, Group root) {
 		super(gameModelAdapter, assets, view);
-		setIsVisible(false);
+		this.root = root;
+		scrollPane.setVbarPolicy(ScrollBarPolicy.ALWAYS);
+		scrollPane.setHbarPolicy(ScrollBarPolicy.NEVER);
+		scrollPane.setContent(structureBox);
+		scrollPane.getStyleClass().setAll("scroll");
+		scrollPane.addEventFilter(ScrollEvent.SCROLL,event -> {
+		    if (event.getDeltaX() != 0) { 
+		        event.consume(); //This disables horizontal scrolling in the scroll pane
+		    }
+		});
+		ds.setOffsetY(3.0);
+		ds.setRadius(3);
+		ds.setColor(Color.color(0, 0, 0));
+		label.setTextFill(Color.WHITE);
+        label.setFont(getAssets().getFont(2));
+        label.setEffect(ds);
+		setUpTable();
+		setUpColumns();
+	}
+
+	private void setUpTable() {
+		scrollPane.addEventFilter(ScrollEvent.SCROLL,event -> {
+		    if (event.getDeltaY() != 0) { 
+		        event.consume(); //This disables vertical scrolling in the actual table
+		    }
+		});
+		structureTable.setEditable(false);
+		structureTable.getStyleClass().addAll("tableViewStyle");
+		structureTable.setItems(structureList);
+		structureBox.setSpacing(5);
+        structureBox.setPadding(new Insets(10, 0, 0, 10));
+        structureBox.getChildren().addAll(label, structureTable);
+	}
+
+	private void setUpColumns() {
+		structureTable.getColumns().add(structureTypeColumn);
```

Comment:
We need to super make sure he's cool with javafx TDA stuff on monday. just a reminder